### PR TITLE
Cleaning up ZO brine concentrator and adding costing

### PR DESCRIPTION
**Pull request by andrewlee94**
## Part of WT3 merger

Adds costing for zero-order brine concentrator models, as well as some general clean up.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by adam-a-a** [comment]
PR# [403](url) adds the crystallizer as a subtype in the yaml file and _should_ have the same cost parameters but with different values. I think 403 has been ready to merge and we just needed one more review. Once merged, we'd need to add the associated cost parameters in the brine_concentrator_zo yaml file

**Review by adam-a-a — APPROVED** [review]
LGTM

**Review by luohezhiming — APPROVED** [review]
LGTM.

**Pull request merged**
